code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -1076,3 +1076,40 @@
 	color = "#efeff0"
 	nutriment_factor = 1.5 * REAGENTS_METABOLISM
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+		// SKYRAT ADDITION BEGIN - Teshari food reagents
+/datum/reagent/consumable/piru_flour
+	name = "Piru Flour"
+	description = "Flour made from ground piru fronds, it seems to clump together with itself."
+	nutriment_factor = 4 * REAGENTS_METABOLISM
+	color = "#c499bc"
+	taste_description = "chalky, sweet flour"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/kiri_jelly
+	name = "Kiri Jelly"
+	description = "Sticky, sweet jelly from ground kiri fruits. It almost seems to sparkle by itself."
+	nutriment_factor = 10 * REAGENTS_METABOLISM
+	color = "#993c5c"
+	taste_mult = 2
+	taste_description = "ultra-sweet jelly"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/muli_juice
+	name = "Muli Juice"
+	description = "A pale-blue liquid contained inside muli pods. Widely used in teshari cuisine."
+	nutriment_factor = 4 * REAGENTS_METABOLISM
+	color = "#6f83a6"
+	taste_mult = 2
+	taste_description = "mintyness"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/nakati_spice
+	name = "Nakati Spice"
+	description = "A coarse brown powder ground from nakati bark which serves as a staple in teshari cooking. The aroma is delightful, but extremely potent."
+	nutriment_factor = 2 * REAGENTS_METABOLISM
+	color = "#735c45"
+	taste_mult = 3
+	taste_description = "potent seasoning"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+		// SKYRAT ADDITION END
```

Comment:
`modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/food_reagents.dm`

Reply:
Gotcha, I'll do that